string piece_type
niryo_robot_msgs/ObjectPose pick_pose
niryo_robot_msgs/ObjectPose place_pose
